src/Handlers/Handler.php

```diff
@@ -157,6 +158,10 @@ public function addParameters(array $parameters): Handler
     public function __invoke(Nutgram $bot): mixed
     {
         try {
+            if (is_string($this->callable) && $this->getInvoker() !== null) {
+                $this->callable = [$this->getInvoker(), $this->callable];
+            }
+
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏this adds extra complexity on of one of the most complicated bits of the framework. The callable must be resolved and set at registration time and not at runtime.
So violates the boundary between steps and makes the Handler mutable